# Bugs

* Replaced id for instance_id on credentials which caused _`$.id` could not be found in the datastructure_ on cluster plans. @itsouvalas